vars/Debian.yml

```
@@ -1,6 +1,15 @@
 ---
 # The system packages to install
+#
+# https://docs.docker.com/engine/install/debian/
 package_names:
+  - containerd.io
   - docker-ce
+  - docker-ce-cli
   - docker-compose-plugin
+  # This package is required to avoid an issue with docker compose
+  # pull.  See, for example:
```

Just adding the issue I created to the comment here.

```suggestion
  # pull.  See the following for more information:
  # - https://github.com/docker/compose/issues/9560
```